### PR TITLE
Fixes #9912: Fix ambiguous column error by using table_name.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
This error is currently also being seen in Katello 2.1 installations and blocks users abilities to successfully install discovery 2.0.1

File: db/migrate/20150310153859_remove_discovery_attribute_sets_from_managed_hosts.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class RemoveDiscoveryAttributeSetsFromManagedHosts < ActiveRecord::Migration
   def up
-    DiscoveryAttributeSet.where(:id => DiscoveryAttributeSet.joins(:host).where(:'hosts.managed' => true).pluck(:id)).delete_all
+    DiscoveryAttributeSet.where(:id => DiscoveryAttributeSet.joins(:host).where(:'hosts.managed' => true).pluck("#{DiscoveryAttributeSet.table_name}.id")).delete_all
```
Comment (ohadlevy) [inline db/migrate/20150310153859_remove_discovery_attribute_sets_from_managed_hosts.rb:3]: :+1: this looks like rails 3.2.8 in scl vs 3.2.13 from gems...

**Comment by orrabin** [comment]
merged as 581c487caa33eed9ec01baa1b656021911fbf2d8, thanks @ehelms!

**Comment by lzap** [comment]
Sorry about that, Debian package was not yet released... :-)

**Comment by lzap** [comment]
Wait it was out too, working on 2.0.2 release now.

**Comment by unorthodoxgeek** [comment]
image

**Comment by mmoll** [comment]
thank-you-gentlemen

**Comment by unorthodoxgeek** [comment]
image

**Pull request closed without merging**